sale_blanket_order/models/blanket_orders.py

```diff
@@ -136,7 +136,9 @@ def _compute_amount_all(self):
     sale_count = fields.Integer(compute="_compute_sale_count")
 
     fiscal_position_id = fields.Many2one(
-        "account.fiscal.position", string="Fiscal Position"
+        "account.fiscal.position",
+        string="Fiscal Position",
+        domain="[('company_id', '=', company_id)]",
```

Comment:
```suggestion
        domain="[('company_id', '=', company_id)]",
        check_company=True,
```
Needed for multi-company consistency check